src/Helper/Fields/Field_Repeater.php

```diff
@@ -107,6 +107,20 @@ public function get_repeater_form_data( $data, $value, $field ) {
 				if ( isset( $form_data['field'] ) ) {
 					$data = array_replace_recursive( $data, [ $id => $form_data['field'] ] );
 				}
+
+				/* Allow HTML to be included in $form_data */
+				if ( isset( $form_data['html_id'] ) ) {
+					$html = [];
+					/* Extract field id and html content */
+					foreach ( $form_data['html_id'] as $field_id => $content ) {
+						/* Format data, with the exact same order ( label.id,id,label ) with the other fields */
+						$html[ $class->field->label . '.' . $field_id ] = $content;
+						$html[ $field_id ]                              = $content;
+						$html[ $class->field->label ]                   = $content;
+					}
+
+					$data = array_replace_recursive( $data, [ $id => $html ] );
```

Comment:
@jestonihpi out of interest, what's the issue with assigning the HTML array keys/value directly to the `$data` array?

Reply:
To make it look like a standard entry array @jakejackson1 iirc. 
If we put it directly into data it would look something like this :
![image](https://user-images.githubusercontent.com/434866/184588241-7179a50e-83b9-482e-843c-33386b823bf9.png)
Where it should be like this :
![image](https://user-images.githubusercontent.com/434866/184588302-d3e5e144-c0b2-49c4-8bfc-4cef21b74369.png)